packages/renderer/src/mux-video-and-audio.ts

```diff
@@ -39,6 +39,8 @@ export const muxVideoAndAudio = async ({
 		videoOutput ? 'copy' : null,
 		audioOutput ? '-c:a' : null,
 		audioOutput ? 'copy' : null,
+		'-movflags',
```

Comment:
I appreciate this isn't ideal because the output codec may not support these flags but it wasn't immediately obvious to me how to check that from this point